examples/src/routes/TimeOverTimeComparison.jsx

```diff
@@ -19,8 +31,51 @@ export const TimeOverTimeComparison = () => (
 
         <hr className="separator" />
 
-        <h2>Comparing to the same period previous year</h2>
-        <ExampleWithSource for={SamePeriodColumnChartExample} source={SamePeriodColumnChartExampleSrc} />
+        <h2>Comparing to the same period previous year Column Chart</h2>
```

Comment:
I would rename it to "Comparing to the same period previous year in column chart" so it can be read nicely as a sentence.
Please, update other examples titles on this page accordingly.